### ISSUE TITLE
a11y

### ISSUE THREAD

**Issue by Rich-Harris** [Rich-Harris/pancake#3]
Right now this library doesn't have anything to say about accessibility. It's potentially in a good place to make sure that people *do* provide relevant information for screen readers and the like, though I need to spend a bit of time educating myself about the various best practices

**Comment by Rich-Harris** [comment]
https://dev.to/mgrsskls/comment/l2cc

**Comment by oneezy** [comment]
Just wanted to add another great a11y resource from @ffoodd who created [**chaarts**](https://github.com/ffoodd/chaarts/):

[Pie Chart Example](https://ffoodd.github.io/chaarts/pie-charts.html)


![image](https://user-images.githubusercontent.com/7369575/152592823-dfc28acd-c070-42d6-9bab-2313d014a21a.png)
